server/eb/app.py

```diff
@@ -62,7 +63,10 @@
     logging.exception("Caught exception during initialization")
```

Comment:
this should be a logging.critical()

Reply:
as should line 75

Reply:
I choose exception to get the exception info into the logs.  I think this will do the same thing for critical:   logging.critical( "Caught exception during initialization", exc_info=True)

Reply:
we definitely want it in the logs...